package/lib/src/utils/text.dart

```diff
@@ -173,6 +173,11 @@ TextStyle textStyleFromJson(ThemeData theme, Map<String, dynamic> json) {
     decorations.add(TextDecoration.lineThrough);
   }
 
+  var height = json["height"];
+  if (height != null && !(height is double)) {
+    height = double.tryParse(height.toString());
+  }
+
```

Comment:
You can use the same approach as for `json["size"]` (see below), can't you?

Reply:
> You can use the same approach as for `json["size"]` (see below), can't you?

complete